dist/index.js

```diff
@@ -153,18 +153,23 @@ function getAIResponse(prompt) {
             top_p: 1,
             frequency_penalty: 0,
             presence_penalty: 0,
+            response_format: { type: "json_object" },
```

Comment:
Since the models we use are often in the standard chat form, `json_object` can be used in the chat form, such as `gpt-4o`, `gpt-4-turbo`, or `gpt-3.5-turbo`.

refer to https://platform.openai.com/docs/guides/text-generation/json-mode